packages/web/src/pages/track-page/components/mobile/TrackHeader.tsx

```diff
@@ -326,17 +327,33 @@ const TrackHeader = ({
         titleMessage = messages.premiumTrack
       }
       return (
-        <div className={cn(styles.typeLabel, styles.gatedContentLabel)}>
-          <IconComponent />
-          <span>{titleMessage}</span>
-        </div>
+        <Flex gap='xs' justifyContent='center' alignItems='center'>
+          <IconComponent color='subdued' size='s' />
+          <Text
+            variant='label'
+            color='subdued'
+            css={{
+              letterSpacing: '2px'
+            }}
+          >
+            {titleMessage}
+          </Text>
+        </Flex>
       )
     }
 
     return (
-      <div className={styles.typeLabel}>
-        {isRemix ? messages.remix : messages.track}
-      </div>
+      <Flex justifyContent='center' alignItems='center'>
+        <Text
+          variant='label'
+          color='subdued'
+          css={{
```

Comment:
with letter spacing... is this just a custom thing from figma for this element?

Reply:
hm actually the designs say 0.5px letter spacing which is the default. i was matching this to the album header letter spacing which is 2px. but maybe it's actually the album one that's wrong.

Reply:
oh interesting, yeah mind confirming designs in figma and updating album header if so?

Reply:
yep designs def say 0.5px